src/EditorFeatures/Core/CommandHandlers/QuickInfoCommandHandlerAndSourceProvider.cs

```diff
@@ -16,65 +16,31 @@
 using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 using Microsoft.VisualStudio.Utilities;
 
-#pragma warning disable CS0618 // IQuickInfo* is obsolete, tracked by https://github.com/dotnet/roslyn/issues/24094
 namespace Microsoft.CodeAnalysis.Editor.CommandHandlers
 {
     [Export]
     [Order(After = PredefinedQuickInfoPresenterNames.RoslynQuickInfoPresenter)]
```

Comment:
Is this Order attribute still needed? From what I can tell, it looks like this previously existed because there were 2 distinct quick info sources for Roslyn (excluding the event hookup case):

1) Quick Info Controller source: this source would listen for quick info computations, cancel the quick info session, and queue an async computation of the quick info items.
2) Quick Info Presenter source: once the async computation of items had completed, this source is updated with a reference to these items, and quick info is invoked via the quick info broker.

As part of your PR, you deleted the Quick Info Presenter source so I think this is no longer needed.

Reply:
If it is no longer needed, you can probably delete ```PredefinedQuickInfoPresenterNames.RoslynQuickInfoPresenter``` too.